docsite/docs/api_pages/face-landmark-detection.md

```diff
@@ -9,4 +9,121 @@ Detects the main points of the face and its expressions in real time.
 For more details or to demo it, visit 
 [MediaPipe - Face Landmark Detection](https://mediapipe-studio.webapps.google.com/studio/demo/face_landmarker)
 
-## How to get started 
\ No newline at end of file
+## How to get started 
+** Using yarn **
+
+**Requirements**
+* Gradle minimum SDK 24 or higher
+* Android-SDK Version 26 or higher
+* iOS 12 or higher
+
+1. **Open Terminal or Command Prompt:** Open your terminal or command prompt application. Create a brand new React Native Project. In this case we will be using **'eyetracker'** as on React Native MediaPipe Demo for example.
+
+```
+npx react-native init eyetracker
+```
+
+2. **Navigate to Your Project Directory:** Navigate to your React Native project directory on your environment.
+
+3. **Install React Native MediaPipe:** Run the following command to install React Native MediaPipe and its dependencies:
+
+```
+yarn add react-native-mediapipe react-native-vision-camera react-native-worklets-core
+```
+
+    If correctly you should see a **react-native-mediapipe** dropdown inside **node_modules** 
```

Comment:
this is not necessary. doing `yarn add` is something that react native developers do every day.